stream-chat-android-client/src/test/java/io/getstream/chat/android/client/clientstate/ClientStateServiceTests.kt

```diff
@@ -241,8 +242,8 @@ internal class ClientStateServiceTests {
 
         fun givenUserConnectedState(
             user: User = Mother.randomUser(),
-            token: String = Mother.randomString(),
-            connectionId: String = Mother.randomString()
+            token: String = randomString(),
```

Comment:
I'd like have one mother object and move it to a common module with JFixture. It's more convenient. What do you think?

Reply:
But I can do it later after merge by my own

Reply:
Agreed, we can take a look at that later on, see if it works